README.asciidoc

```diff
@@ -1,4 +1,4 @@
-= os-autoinst image:https://api.travis-ci.org/os-autoinst/os-autoinst.svg?branch=master[link=https://travis-ci.org/os-autoinst/os-autoinst] image:https://codecov.io/gh/os-autoinst/os-autoinst/branch/master/graph/badge.svg[link=https://codecov.io/gh/os-autoinst/os-autoinst]
+= os-autoinst image:https://github.com/os-autoinst/os-autoinst/workflows/ci/badge.svg?branch=master[link=https://github.com/os-autoinst/os-autoinst/actions] image:https://codecov.io/gh/os-autoinst/os-autoinst/branch/master/graph/badge.svg[link=https://codecov.io/gh/os-autoinst/os-autoinst]
```

Comment:
Looks nice:

![image](https://user-images.githubusercontent.com/1204189/102994186-e21fc380-451e-11eb-8ff9-c105ad0a9dc7.png)

Although it seems this PR is based on #1602 for some reason

Reply:
#1602 is merged meanwhile so the inheritance is not a problem anymore. I have for now concentrated on documentation changes only for this PR.